src/Serializer/Serializable.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Serializer;
+
+interface Serializable
+{
+    /**
+     * Return an array representation of the object for Sentry.
+     *
+     * @return array
+     */
+    public function __toSentry(): array;
```

Comment:
I'm against the use of the double underscore for the name, since it make it seems to say that this is a "magic method", which is not.

IMHO this is fine:
```suggestion
    public function toSentry(): array;
```

Reply:
Agreed! 👍